pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -314,6 +314,12 @@ pyramid_oereb:
         methods:
           date: pyramid_oereb.standard.hook_methods.get_surveying_data_update_date
           provider:  pyramid_oereb.standard.hook_methods.get_surveying_data_provider
+    general_information:
+      text:
```

Comment:
This line has to been removed to get it working, otherwise the dict hirarchy does not work out for me...
I'm amending that in my french version, ok?